### ISSUE TITLE
Use dialog element

### ISSUE THREAD

**Issue by gavmck** [reactjs/react-modal#212]
Could this make use of the dialog element for greater accessibility?

http://www.w3schools.com/tags/tag_dialog.asp

**Comment by diasbruno** [comment]
@gavmck It would be nice, but it seems to be implemented in a few browsers...
There is an attribute `role=dialog`, maybe it can be useful for now.

Sources:
[MDN - Dialog](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_dialog_role)
 [caniuse.com/dialog](http://caniuse.com/#feat=dialog)

**Comment by NickHeiner** [comment]
Yes, you should be putting `role="dialog"` on the modal element. That will assist screen readers by letting them understand the semantic role of the element in the app.

**Comment by NickHeiner** [comment]
Nice!

**Comment by gavmck** [comment]
@OliverJAsh That would be great :rocket:

**Comment by jaxor24** [comment]
Also keen for this
